### ISSUE TITLE
Villager trading global bookmark bug

### ISSUE THREAD

**Issue by Taro010** [blackd/Inventory-Profiles#332]
### Have you asked for help on discord?

NO

### Minecraft Version/s

1.19.4

### Modloader

Fabric

### Modloader Version

Fabric loader 0.14.21
Fabric API 0.76.0

### List of all other installed mods

fabric-api-0.76.0+1.19.4
fabric-language-kotlin-1.9.4+kotlin.1.8.21
InventoryProfilesNext-fabric-1.19.4-1.10.2
libIPN-fabric-1.19.4-3.0.1

### Describe the problem

https://github.com/blackd/Inventory-Profiles/assets/105798475/853bf2f2-28df-40d0-adfd-02ca81cf297d

I cannot bookmark the last trade slot of the villager

### Steps to reproduce

So basically, you make a trading villager and full their trade gate then try bookmark the last slot, you will find that you cannot interact it, but the first till the almost the last work perfectly, the only issue is the last slot can't not been mark.

### I have tried to reproduce the problem with only or without "Inventory Profiles Next" and its dependencies installed.

- [X] I have tested without 'Inventory Profiles Next' and its dependencies
- [X] I have tested with only 'Inventory Profiles Next' and its dependencies and no other mods

**Comment by blackd** [comment]
Damn I thought this is fixed in 1.10.2

try the shortcuts B for global L for local

**Comment by Taro010** [comment]
> Damn I thought this is fixed in 1.10.2
> 
> try the shortcuts B for global L for local

OMG when I press the shortcut it's work!!!!

**Comment by blackd** [comment]
fixed in 1.10.3
